### PR TITLE
DWR-1592 Adjust validation scripts to support OLAP IAB

### PR DESCRIPTION
**Pull request by agorina**
I ran it on QA and it validates (with the exception of a known rounding scale score issue).

### CODE REVIEW

File: validation/README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ reporting_olap_password=
 ```bash
 ./validate-migration.sh secrets/local.sh
 ```
-2. To validate both reporting only
```
Comment (agorina) [inline validation/README.md]: Unrelated, fixed a typo.

File: validation/sql/total-iab-by-asmt-schoolyear-condition-complete/olap_reporting.sql [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    e.school_year,
+    ac.code,
+    CASE WHEN completeness_id = 2 THEN 'TRUE' ELSE 'FALSE' END
+  FROM fact_student_iab_exam e
+    JOIN administration_condition ac ON e.administration_condition_id = ac.id
+  GROUP BY
+    e.asmt_id,
+    e.school_year,
+    ac.code,
+    completeness_id
```
Comment (jtreuting) [inline validation/sql/total-iab-by-asmt-schoolyear-condition-complete/olap_reporting.sql]: nit: without alias not sure which table this is from

File: validation/sql/total-iab/olap_warehouse.sql [hunk 3]
```diff
@@ -0,0 +1,12 @@
+select count(*) from (
+  SELECT DISTINCT
+    school_year,
+    asmt_id,
+    student_id
+  FROM exam e1
+  where type_id = 2
+        AND deleted = 0
+        AND e1.scale_score IS NOT NULL
```
Comment (jtreuting) [inline validation/sql/total-iab/olap_warehouse.sql]: nit: mix of alias and not

File: validation/sql/total-iab-by-asmt-schoolyear-condition-complete/olap_reporting.sql (hunk 2 above)
Comment (agorina) [reply validation/sql/total-iab-by-asmt-schoolyear-condition-complete/olap_reporting.sql]: Will fix, here and in the source I used to copy it from.

File: validation/sql/total-iab/olap_warehouse.sql (hunk 3 above)
Comment (agorina) [reply validation/sql/total-iab/olap_warehouse.sql]: ditto.

**Pull request merged**